pkg/descheduler/strategies/nodeutilization/nodeutilization.go

```diff
@@ -345,12 +385,16 @@ func isNodeWithLowUtilization(usage NodeUsage) bool {
 	return true
 }
 
-// getResourceNames returns list of resource names in resource thresholds
-func getResourceNames(thresholds api.ResourceThresholds) []v1.ResourceName {
-	resourceNames := make([]v1.ResourceName, 0, len(thresholds))
+// getResourceNames returns list of resource names in resource thresholds, and add the default ones
+func getResourceNames(thresholds api.ResourceThresholds) (resourceNames []v1.ResourceName) {
```

Comment:
`thresholds` is expected to have all the resources listed. Including CPU, Memory and Pods. No need to extent the function with those.